themes/tiddlywiki/vanilla/base.tid

```diff
@@ -92,7 +91,7 @@ pre {
 	white-space: pre-wrap;
 	background-color: <<colour pre-background>>;
 	border: 1px solid <<colour pre-border>>;
-	padding: 0 3px 2px;
+	padding: 0.5em;
```

Comment:
> this PR is not the place to kind of sneak this change into the core

> Using extra markup to change defaults is not the way to go.

@tobibeer, Do I understand you correctly: you are in favor of changing the default pre padding, but think vanilla/base.tid is not the place to do it? Where else? That's where it's currently defined.